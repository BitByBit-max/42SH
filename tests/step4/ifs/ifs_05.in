var="toto | cat -e | cat -n"

IFS='|';for i in "$var"; do
	echo $i; echo "$i"
done
