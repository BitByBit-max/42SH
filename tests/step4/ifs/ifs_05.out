toto   cat -e   cat -n
toto | cat -e | cat -n
